frame/utility/src/lib.rs

```diff
@@ -195,6 +196,58 @@ decl_module! {
 			origin.set_caller_from(frame_system::RawOrigin::Signed(pseudonym));
 			call.dispatch(origin).map(|_| ()).map_err(|e| e.error)
 		}
+
+		/// Send a batch of dispatch calls and atomically execute them.
+		/// The whole transaction will rollback and fail if any of the calls failed.
+		///
+		/// May be called from any origin.
+		///
+		/// - `calls`: The calls to be dispatched from the same origin.
+		///
+		/// If origin is root then call are dispatch without checking origin filter. (This includes
+		/// bypassing `frame_system::Trait::BaseCallFilter`).
+		///
+		/// # <weight>
+		/// - Base weight: 14.39 + .987 * c µs
```

Comment:
We can remove this line and any other similar lines from the pallet

Reply:
It should instead say this function is `O(c)` where C is the number of calls to be batched